internal/backend/sessions/sessioncalls/calls.go

```diff
@@ -32,10 +38,13 @@ type Calls interface {
 }
 
 type calls struct {
-	z      *zap.Logger
-	config Config
-	worker worker.Worker
-	svcs   *sessionsvcs.Svcs
+	z             *zap.Logger
+	config        Config
+	generalWorker worker.Worker // general worker that can serve any workflows.
+	uniqueWorker  worker.Worker // unique worker that can server workflows that run only this this process.
+	svcs          *sessionsvcs.Svcs
+
+	uniqueWorkerCallTaskQueueName string
```

Comment:
What is a `uniqueWorker`? The fact we have two different types of worker is a `design smell`.

Reply:
commented